drivers/ieee802154/Kconfig.nrf5

```diff
@@ -11,12 +11,6 @@ menuconfig IEEE802154_NRF5
 
 if IEEE802154_NRF5
 
-config IEEE802154_NRF5_DRV_NAME
```

Comment:
So how can be a device name specified now, I've tried to check what is the name actually assigned with those changes, but it seems to be an empty string.

I'm asking, as the network device name is particularly important in terms of networking, as it also identifies individual network interfaces, w/o a name application is not able to use `SO_BINDTODEVICE` socket option for example.

Reply:
I guess one can always use `DEVICE_DT_NAME_GET`. @galak any comments here? it may be relevant for the `label` removals.